warehouse/manage/views.py

```diff
@@ -1534,7 +1534,7 @@ def _error(message):
             self.request.route_path(
                 "manage.project.release",
                 project_name=self.release.project.name,
-                version=self.release.version,
```

Comment:
I believe there are other redirects in this module that should have the same change?